packages/web/components/Form/FormWizard.js

```diff
@@ -165,7 +169,28 @@ const FormWizard = ({ steps, currentStep, onSubmit, onPrev, data, defaultValues,
 	 * @param {object} form A instance of the `useForm` hook.
 	 */
 	const handleSubmit = (formData, form) => {
-		onSubmit({ data: formData, step: currentStepSlug, nextStep }, form);
+		let formattedData = formData;
+		if (currentStepSlug === 'costs') {
+			const { funding_value } = formData.technologyCosts;
+
+			formattedData = {
+				...formData,
+				technologyCosts: {
+					...formData.technologyCosts,
+					costs: {
+						implementation_costs: parseCostValueToInt(
+							formData.technologyCosts.costs.implementation_costs,
```

Comment:
We have to safeguard this code here to avoid breaking the app in case the `technologyCosts.costs` does not exist (case the user taps the continue button).

![Screen Shot 2020-10-27 at 06 22 10](https://user-images.githubusercontent.com/12154623/97281871-c8167d00-181c-11eb-824a-555ac45acbda.png)

Reply:
Ty for reaching this, @lcnogueira. Great QA 🤝 

It's done now